Examples/ObjectiveC/RuntimeMultipleAnnotationsExample.m

```diff
@@ -49,32 +49,40 @@ - (void)addItemsToMap:(NSArray *)features {
     // In this case, we can use style functions to gradually change properties between zoom level 2 and 7: the circle opacity from 50% to 100% and the circle radius from 2pt to 3pt.
     MGLCircleStyleLayer *circles = [[MGLCircleStyleLayer alloc] initWithIdentifier:@"lighthouse-circles" source:source];
     circles.circleColor = [MGLStyleValue valueWithRawValue:lighthouseColor];
-    circles.circleOpacity = [MGLStyleValue valueWithStops:@{
-        @2: [MGLStyleValue valueWithRawValue:@0.5],
-        @7: [MGLStyleValue valueWithRawValue:@1.0],
-    }];
-    circles.circleRadius = [MGLStyleValue valueWithStops:@{
-        @2: [MGLStyleValue valueWithRawValue:@2],
-        @7: [MGLStyleValue valueWithRawValue:@3],
-    }];
-
+    circles.circleOpacity = [MGLStyleValue valueWithInterpolationMode:MGLInterpolationModeExponential
+        cameraStops : @{
+                        @2: [MGLStyleValue valueWithRawValue:@0.5],
+                        @7: [MGLStyleValue valueWithRawValue:@1.0]
+                        }
+        options: @{MGLStyleFunctionOptionDefaultValue : [MGLStyleValue valueWithRawValue:@0.75]}];
+                             
+    circles.circleRadius = [MGLStyleValue valueWithInterpolationMode:MGLInterpolationModeInterval cameraStops:@{
```

Comment:
I suspect a line this long will cause it to wrap on our website. Either way, let’s keep the line breaks for `cameraStops` consistent.